packages/nextjs/components/scaffold-stark/BlockExplorer.tsx

```diff
@@ -4,6 +4,8 @@ import { Address as AddressType, mainnet } from "@starknet-react/chains";
 import { MagnifyingGlassIcon } from "@heroicons/react/24/outline";
```

Comment:
So far looks good ! Kindly check also the button in the faucet when the network is set to `devnet`. Would be great for it to have the same style as well (still in scope of this task).

![Uploading image.png…]()

Reply:
Hi @stephanniegb, I agree with @metalboyrick. A review of the local faucet may be very helpful, but I really like how it turned out; it maintains a consistent style across the different components.

The light tone does not differentiate well among the various options; it lacks migrate the gradient border color.

![image](https://github.com/user-attachments/assets/c121ebc1-fe43-4c78-ab0e-e4c7c1639cd2)

Reply:
Okay, I will work on them